tableaudocumentapi/column.py

```diff
@@ -0,0 +1,62 @@
+import functools
+
+_ATTRIBUTES = [
```

Comment:
Can you comment how each of these fields relates to the UI, it's not intuitive that caption is what most people would call the 'name' in the UI, etc.